ffi/android/Dockerfile

```diff
@@ -0,0 +1,60 @@
+FROM rustlang/rust:nightly-slim as builder
```

Comment:
BTW, can we stick to a specific nightly version here? We had experience of nightly versions breaking before

Reply:
[rustlang/rust](https://hub.docker.com/r/rustlang/rust/) does not keep track of specific versions, it just uses `nightly` and `nightly-slim` and overrides them over time.

[rust](https://hub.docker.com/_/rust), which should be better since it contains the official docker images for rust, unfortunately does not have the latest releases.

for this project we should need to use `rust:1.49-slim-buster` but the latest available tag is `1.47-slim-buster` (which is too old for this project, it gives `E0554` at build time). so we can open an issue and do it as soon as they publish an updated image.

Reply:
Fortunately this night I was able to make both LNP/BP Core Lib and RGB Node compiling on rust down to 1.41.1, so we can switch to normal versions like I did here https://github.com/LNP-BP/docker/commit/0aad29ed364169a1d8502e2139930eb3a5fbb3fb#diff-d4c727902165b154b499e32ea11043fbf11b2b5e9c285a1d2f2be3863f5363a6 and here https://github.com/LNP-BP/rgb-node/commit/4502981bbf3ce1538fb6fc8acf82cadbd2c5ef16

Reply:
I updated the docker base image, but to avoid the `E0554` error I had to run a `cargo update`.

after doing that, I've received this error on build:
```
#15 420.3    Compiling rgb_node v0.1.0 (https://github.com/LNP-BP/rgb-node?branch=master#4502981b)
#15 433.1 error[E0061]: this function takes 6 arguments but 7 arguments were supplied
#15 433.1    --> src/lib.rs:236:10
#15 433.1     |
#15 433.1 236 |         .transfer(
#15 433.1     |          ^^^^^^^^ expected 6 arguments
#15 433.1 237 |             data.inputs,
#15 433.1     |             -----------
#15 433.1 238 |             data.allocate,
#15 433.1     |             -------------
#15 433.1 239 |             data.invoice,
#15 433.1     |             ------------
#15 433.1 240 |             data.prototype_psbt,
#15 433.1     |             -------------------
#15 433.1 241 |             data.fee,
#15 433.1     |             --------
#15 433.1 242 |             data.consignment_file,
#15 433.1     |             ---------------------
#15 433.1 243 |             data.transaction_file,
#15 433.1     |             --------------------- supplied 7 arguments
#15 433.1 
#15 433.2 error: aborting due to previous error
```
so I've fixed transfer signature.

but then on build I've received this error:
```
#15 464.8   = note: /usr/bin/ld: cannot find -lsqlite3
#15 464.8           collect2: error: ld returned 1 exit status
#15 464.8           
#15 464.8 
#15 464.8 error: aborting due to previous error
#15 464.8 
#15 464.8 error: could not compile `rgb`.
```

I tried to add `libsqlite3-dev` to the docker image, but still receiving the error (a little different from before):
```
#15 484.0   = note: /root/sdk/ndk/20.1.5948944/toolchains/llvm/prebuilt/linux-x86_64/bin/../lib/gcc/aarch64-linux-android/4.9.x/../../../../aarch64-linux-android/bin/ld: cannot find -lsqlite3
#15 484.0           clang: error: linker command failed with exit code 1 (use -v to see invocation)
#15 484.0           
#15 484.0 
#15 484.0 error: aborting due to previous error; 2 warnings emitted
#15 484.0 
#15 484.0 error: could not compile `rgb`.
```

therefore I need to debug the situation a little bit more

Reply:
fixed by adding this to `Cargo.toml`:
```
[dependencies.libsqlite3-sys]
version = "0.9.1"
features = ["bundled"]
```

Reply:
Yeah, these issues were related to https://github.com/LNP-BP/rgb-sdk/issues/8